.eslintrc

```
@@ -18,7 +18,7 @@
     "plugin:prettier/recommended"
   ],
   "parserOptions": {
-    "ecmaVersion": 2018,
+    "ecmaVersion": 2020,
```

```suggestion
    "ecmaVersion": 2018,
```
Can we change the ecma version back to `2020` and instead only disable eslint for the lines where we use dynamic imports?